src/css/styles.css

```diff
@@ -345,3 +319,10 @@ iframe {
 .login-form__footer {
     text-align: right;
 }
+
+.loading {
+  margin-left: 20px;
+  height: 70px;
+  background-image: url(../img/infinity-loader.svg);
+  background-repeat: no-repeat;
```

Comment:
Can we add `background-position-x: center` instead of using `margin-left`? Just preferred aesthetically.